packages/terra-form-input/src/terra-dev-site/doc/example/uncontrolled/DefaultExample.jsx

```diff
@@ -4,11 +4,11 @@ import Input from 'terra-form-input';
 
 const DefaultExample = () => (
   <div>
-    <Field label="Default" htmlFor="uncontrolled-default">
+    <Field label="Default">
       <Input
         name="default input"
-        id="uncontrolled-default"
         defaultValue="Default Input – Uncontrolled"
+        ariaLabel="form-input-label"
```

Comment:
Let's update this to the following:
```suggestion
        ariaLabel="Default"
```